src/test.rs

```diff
@@ -507,6 +507,10 @@ impl TestRequest<()> {
     {
         TestRequest::default().header(key, value)
     }
+    /// Create TestRequest and set request cookie
```

Comment:
Add empty line before this doc string

Reply:
Got it. 👍 
Maybe we need a contribution guide for creating a pull request or a feature so that we can do all the things we need at one time, FYI.

Reply:
@DoumanAsh All done. And, should I add a tag named "v0.7.18"?